fpgalib/adc.py

```diff
@@ -709,26 +677,29 @@ def makeTriggerTable(cls, triggerTable, p):
             raise Exception("Trigger table max len = 128")
         if triggerTable[0][1] < 1:
             raise Exception("Trigger table row0 rdelay is 0")
-        
+
         rlens = [row[1]<50 for row in triggerTable]
-        
-        # if there is a spacing of <50 clock cycles between demods then the FIFO gets backed up
+
+        # if there is a spacing of <50 clock cycles between demods then the FIFO
+        # gets backed up
         if any(rlens):
             count0 = triggerTable[0][0]
             rlen0 = triggerTable[0][1]
-            if rlen0<50 and count0==1 and not any(rlens[1:]): # if its only the start delay, no exception
+            # if its only the start delay, no exception
+            if rlen0<50 and count0==1 and not any(rlens[1:]):
                 pass
             else:
-                raise Exception("rlen < 50 clock cycles (200 ns) can cause FIFO backup for 12 chans")
-        
+                raise Exception("rlen < 50 clock cycles (200 ns) can cause FIFO"
+                                "backup for 12 chans")
```

Comment:
You'll want a space in one of these string literals, because they get concatenated without any whitespace:
```python
raise Exception("rlen ... cause FIFO "
                "backup for 12 chans")